src/components/AdminSettings.vue

```diff
@@ -90,6 +90,17 @@
                     for="sso_force_iframe"
                 >{{ t('riotchat', 'Disable redirect to non-iframed version for SSO (make sure to set the headers to allow the SSO or CAS to be iframed)') }}</label>
                 <br>
+                <label
+                    ref="sso_iframe_domain"
+                    for="sso_iframe_domain"
+                >{{ t('riotchat', 'External domains allowed to be iframed:') }}</label>
```

Comment:
We should have a note here about the input being space-delimited. Probably not a a blocking issue since most are likely to use this with only one domain.